### PR TITLE
Update the game over message

### PR DESCRIPTION
**Pull request by Lily1310**
Update the game over message so people know how to play again!!

### CODE REVIEW

**Review by Lily1310 — COMMENTED** [review]
A **pull request** (PR) is a way to propose changes to a code repository. It allows developers to:

1. **Submit Code Changes:** When a developer finishes working on a feature or fix, they create a pull request to submit their changes for review.

2. **Review Process:** Other team members can review the changes, discuss them, and suggest improvements.

3. **Merge Changes:** Once approved, the changes can be merged into the main codebase, making them part of the project.

In short, a pull request facilitates collaboration and code quality in software development.

File: index.html [hunk 1]
```diff
@@ -47,7 +47,7 @@ <h1>Tic Tac Toe</h1>
   </tr>
 </table>
 
-<h2 hidden>Game over</h2>
+<h2 hidden>Game over, refresh to play again 🧑‍💻 🤖!</h2>
```
Comment (Lily1310) [inline index.html:50]:
```suggestion
<h2 hidden>Game over, refresh to play again 🧑‍💻 🤖!</h2>
Editing the suggestion from the task
```
